### PR TITLE
LOC CHECKIN | dotnet/roslyn master-vs-deps | 20190621

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Workspaces/Core/Portable/xlf/WorkspacesResources.fr.xlf [hunk 1]
```diff
@@ -641,7 +641,7 @@ L'ancre $ correspond à \n mais ne correspond pas à \r\n (combinaison de caract
       </trans-unit>
       <trans-unit id="Regex_match_zero_or_one_time_long">
         <source>The ? quantifier matches the preceding element zero or one time. It is equivalent to the {0,1} quantifier. ? is a greedy quantifier whose lazy equivalent is ??.</source>
-        <target state="translated">Le quantificateur ? recherche une correspondance avec l'élément précédent, zéro ou une fois. Il est équivalent au quantificateur {0,1}. ? est un quantificateur gourmand dont l'équivalent paresseux est ??.</target>
+        <target state="translated">Le quantificateur ? recherche une correspondance avec l'élément précédent, zéro ou une fois. Il est équivalent au quantificateur {0,1,}. ? est un quantificateur gourmand dont l'équivalent paresseux est ??.</target>
```
Comment (tmeschter) [inline src/Workspaces/Core/Portable/xlf/WorkspacesResources.fr.xlf]: That should be `{0,1}` not `{0,1,}`.

Comment (tmeschter) [reply src/Workspaces/Core/Portable/xlf/WorkspacesResources.fr.xlf]: Fixed

**Comment by janbaltus** [comment]
@tmeschter please see updated PR with fixed issue

**Comment by tmeschter** [comment]
@jinujoseph This loc PR is ready for approval.

**Pull request merged**